Gem/Code/Source/AutoGen/NetworkWeaponsComponent.AutoComponent.xml

```diff
@@ -16,6 +16,7 @@
 
     <NetworkInput Type="bool" Name="Draw" Init="false" />
     <NetworkInput Type="WeaponActivationBitset" Name="Firing"  Init="" />
+	<NetworkInput Type="AZ::Vector3" Name="FireTranslation" Init="" Container="Array" Count="MaxWeaponsPerComponent"/>
```

Comment:
Indentation

Reply:
nit: Can we think of a better name than FireTranslation, this is a translation in reference to what? I don't have a good suggestion, but this is definitely a name I would have to look up in documentation based on current naming.